### PR TITLE
add example and update messaging

### PR DESCRIPTION
**Pull request by builtbylane**

### CODE REVIEW

**Comment by shinnn** [comment]
Test fails. https://travis-ci.org/shinnn/isogram/builds/56951927

**Comment by builtbylane** [comment]
will fix now

**Comment by shinnn** [comment]
@builtbylane You don't need to close this PR. Just force push to the `master` branch and the PR will be updated.

**Comment by builtbylane** [comment]
ok, thanks

File: README.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ These are real examples using the code *isogram* generates.
 | gulp.js                       | [gulpjs.com][gulp]        | `g` `u` `l` `p` `j` `s`     |
 | hapi                          | [hapijs.com][hapi]        | `h` `a` `p` `i` `j` `s`     |
 | Jekyll                        | [jekyllrb.com][jekyll]    | `j` `e` `k` `y` `l` `L`     |
+| Jekyll                        | [kanyewest.com][builtbylane]    | `k` `a` `n` `y` `e`     |
```
Comment (shinnn) [inline README.md]: Currently your PR uses wrong markdown syntax so it cannot be shown as you expected. https://github.com/shinnn/isogram/tree/31042992f81a916c0b6cb3e1f5e4fee5de221de9#sites-using-isogram Please update it to be consistent with other links.

**Comment by builtbylane** [comment]
new to the world of testing :blush:. learning a lot from your code.

**Comment by shinnn** [comment]
Sorry to bother you, @jonschlinkert. I'm not a native English speaker so I'd like to hear your opinion :)

When I want to express `3 <= x <= 7` in English, is "no fewer than 3 and no greater than 7"
more natural than "3 and more or 7 and less"?

(This doesn't mean I don't rely on @builtbylane. I just want another opinion.)

**Comment by shinnn** [comment]
@builtbylane Thanks for your great contribution!

Would you mind squashing those 6 commits to 2 commits like below?

1st: `add kanyewest.com to the showcase`
2nd: `fix grammar`

If you don't have enough time to do this, I'll squash them by myself.

File: README.md (hunk 1 above)
Comment (shinnn) [inline README.md]: Really nice fix. https://github.com/builtbylane/isogram/commit/be8a59deb9d92d6fc0ad71294d155974ebda0b3d

**Comment by jonschlinkert** [comment]
> no fewer than 3 and no greater than 7

Yes, that would be more fluent english grammar. 

I'm flattered you asked! happy to help anytime

**Comment by shinnn** [comment]
:+1: Thanks again @builtbylane and @jonschlinkert!

**Pull request merged**